bridge/client/app/_views/ktb-sequence-view/ktb-sequence-view.component.ts

```diff
@@ -91,64 +91,68 @@ export class KtbSequenceViewComponent implements OnInit, OnDestroy {
   constructor(private _changeDetectorRef: ChangeDetectorRef, private dataService: DataService, private route: ActivatedRoute, public dateUtil: DateUtil, private router: Router, private location: Location) { }
 
   ngOnInit() {
-    this.currentSequence = null;
-    this.route.params
-      .pipe(takeUntil(this.unsubscribe$))
-      .subscribe(params => {
+    const projectName$ = this.route.params
+      .pipe(
+        map(params => params.projectName)
+      );
+    this.roots$ = this.dataService.roots;
+    const roots$ = this.dataService.roots
```

Comment:
What's the difference between `this.roots$` and `const roots$` and can we combine those to prevent confusion?

Reply:
The difference is that the second one skips empty roots. This is necessary because later I set the parameters for this page and only take 1. Because we use `BehaviorSubject` the initial value is also emitted